homeassistant/components/risco/binary_sensor.py

```diff
@@ -38,6 +91,15 @@ async def async_setup_entry(
                 RiscoLocalArmedBinarySensor(local_data.system.id, zone_id, zone),
             )
         )
+
+        system_entities = (
+            RiscoSystemBinarySensor(
+                local_data.system.id, local_data.system.system, entity_description
+            )
+            for entity_description in SYSTEM_ENTITY_DESCRIPTIONS
+        )
+
+        async_add_entities(chain(system_entities, zone_entities))
```

Comment:
We usually just create one list and extend that and pass it to the function. Not a blocker, just wanted to let you know